lib/Cake/Test/Case/TestSuite/CakeTestCaseTest.php

```diff
@@ -344,7 +344,7 @@ public function testAssertTextStartsWith() {
  */
 	public function testAssertTextStartsNotWith() {
 		$stringDirty = "some\nstring\r\nwith\rdifferent\nline endings!";
-		$stringClean = "some\nstring\nwith\ndifferent\nline endings!";
+		// $stringClean = "some\nstring\nwith\ndifferent\nline endings!";
```

Comment:
Because it is an unused variable but I assumed it stands there for documentation purposes. Dunno.